public/index.html

```diff
@@ -2,6 +2,23 @@
 <html>
   <head></head>
   <body>
+    <script type="text/javascript" id='courseContext'>
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-var */
+/* eslint-disable quotes */
+var courseContext = {
+  lang: "fr",
+  url_name: "course",
+  name: "edX Demonstration Course",
+  source_of_data: "edx-platform/cms/templates/asset_index.html",
+  display_course_number: "",
+  commit_sha: "30ea0890f4",
+  num: "DemoX",
+  org: "edX",
+  id: "course-v1:edX+DemoX+Demo_Course",
+  revision: "",
+};
```

Comment:
This is set up with the default course settings to be able to test within the SFE container locally

Reply:
I can easily add it into this PR along with the comment -- should the `base_url` be the localhost? I.e `http://localhost:18010`?